.github/workflows/asv.yml

```diff
@@ -78,17 +78,36 @@ jobs:
       run: |
         pip install --upgrade pip
         pip install -e .
-        pip install asv --upgrade
+        pip install --upgrade asv
```

Comment:
this whole file changing -- this is a rebase mess up. We need to clean it up, it's possible that you were rebasing on top of your local master when your local master wasn't up-to-date with the remote master.

Reply:
should be fixed now